Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -830,6 +830,13 @@ public async Task Reload( CancellationToken cancellationToken = default )
             }
         }
 
+        /// <summary>
+        /// Notifies the <see cref="DataGrid{TItem}"/> that state has changed.
+        /// </summary>
+        /// <returns></returns>
+        protected internal new void  StateHasChanged()

```

Comment:
Instead of overriding StateHasChanged and making it accessible from outside I would have a `Task Refresh()` that would call `InvokeAsync( StateHasChanged );`

Reply:
Good idea. Will change.